### DISCUSSION TITLE
Configure a client, or group of clients to use different block lists

### DISCUSSION THREAD

**Discussion by ljlongwing** [TechnitiumSoftware/DnsServer#539] [Ideas]
I'm moving from Pi-Hole to Technitium DNS.  One thing I'm looking for, and I don't know if it's implemented, just differently, is the ability to configure specific machines to bypass the blocking.

Here is my scenario.  I want most machines to block ads, but one person on my network enjoys clicking on those ads and doesn't like getting dns errors when they try to go to those blocked locations.

In Pi-Hole you could define 'groups', and each block list can be assigned to groups.  I created a group for this person, and that group didn't have any block lists assigned to it...so, for anyone in that group (them), no blocking occurred...but DNS worked without issue...for the rest of the network they were in the default group and they got their ads blocked seamlessly.

So....hopefully, understanding my question, is there a way to configure this in your DNS server?

**Comment by Hemsby** [comment]
You can use the advanced Blocking App to achieve this.

Get Outlook for <https://aka.ms/ghei36> Android<https://aka.ms/ghei36>
________________________________
From: ljlongwing ***@***.***>
Sent: Wednesday, January 25, 2023 10:25:09 PM
To: TechnitiumSoftware/DnsServer ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [TechnitiumSoftware/DnsServer] Configure a client, or group of clients to use different block lists (Discussion #539)


I'm moving from Pi-Hole to Technitium DNS. One thing I'm looking for, and I don't know if it's implemented, just differently, is the ability to configure specific machines to bypass the blocking.

Here is my scenario. I want most machines to block ads, but one person on my network enjoys clicking on those ads and doesn't like getting dns errors when they try to go to those blocked locations.

In Pi-Hole you could define 'groups', and each block list can be assigned to groups. I created a group for this person, and that group didn't have any block lists assigned to it...so, for anyone in that group (them), no blocking occurred...but DNS worked without issue...for the rest of the network they were in the default group and they got their ads blocked seamlessly.

So....hopefully, understanding my question, is there a way to configure this in your DNS server?

**Reply by ljlongwing** [reply]
I installed that app and think I have it configured now....I'll check with them to verify that they aren't being blocked now...thank you for the prompt response
